app/src/main/java/com/fstyle/structure_android/data/source/remote/config/service/NameServiceClient.java

```diff
@@ -1,7 +1,8 @@
-package com.fstyle.structure_android.data.source.remote_api.service;
+package com.fstyle.structure_android.data.source.remote.config.service;
 
 import android.app.Application;
 import android.support.annotation.NonNull;
+import com.fstyle.structure_android.data.source.local.config.sqlite.UserDbHelper;
```

Comment:
ERROR: Unused import: com.fstyle.structure_android.data.source.local.config.sqlite.UserDbHelper. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports